commons/src/test/scala/dagr/commons/util/CaptureSystemStreams.scala

```diff
@@ -21,7 +21,7 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  * THE SOFTWARE.
  */
-package dagr.core.util
+package dagr.commons.util
```

Comment:
Since this class is only used in tests, perhaps it should be in the test source path?